packages/ripple/src/runtime/internal/client/runtime.js

```diff
@@ -563,6 +563,12 @@ function flush_queued_root_blocks(root_blocks) {
   }
 }
 
+// https://github.com/sveltejs/svelte/blob/ded13b825d7efcdf064fd65a5aa9e7e61293a48b/packages/svelte/src/internal/client/runtime.js#L501
+export async function tick() {
+  await Promise.resolve();
+  flush_sync();
+}
```

Comment:
```suggestion
/**
 * @returns {Promise<void>}
 */
export async function tick() {
	return new Promise((f) => requestAnimationFrame(() => f()));
	flushSync();
}
```

since we have try / pending and allow awaits in components